xml/security_preface.xml

```diff
@@ -8,753 +8,450 @@
-     relationship with other hosts, based on IP addresses or host names. The
-     attacker needs a good understanding of the actual structure of the
-     trust relationships among hosts to disguise itself as one of the
-     trusted hosts. Usually, the attacker analyzes some packets received
-     from the server to get the necessary information. The attacker often
-     needs to target a well-timed DoS attack at the name server as well.
-     Protect yourself by using encrypted connections that can verify
-     the identity of the hosts to which to connect.
+     Use OpenVPN to secure communication channels over insecure
+     physical networks. For details, see <xref
+     linkend="cha.security.vpnserver" />.
     </para>
-   </sect3>
-   <sect3 xml:id="sec.security.general.overview.network.worms">
-    <title>Worms</title>
+   </listitem>
+   <listitem>
     <para>
-     Worms are often confused with viruses, but there is a clear difference
-     between the two. Unlike viruses, worms do not need to infect a host
-     program to live. Instead, they are specialized to spread as quickly as
-     possible on network structures. The worms that appeared in the past,
-     such as Ramen, Lion, or Adore, used well-known security holes in
-     server programs like bind8. Protection against worms is relatively
-     easy. Given that some time elapses between the discovery of a security
-     hole and the moment the worm hits your server, there is a good chance
-     that an updated version of the affected program is available on time.
-     That is only useful if the administrator actually installs the security
-     updates on the systems in question.
+     Use strong authentication for network services. For details, see
+     <xref linkend="part.auth" />.
     </para>
-   </sect3>
-  </sect2>
+   </listitem>
+  </itemizedlist>
  </sect1>
- <sect1 xml:id="sec.security.general.tips">
-  <title>Some General Security Tips and Tricks</title>
-
-  <remark condition="clarity"> 2007-07-27 - jjaeger: please use profiling for mailing lists and
-   websites if they change from SLE to opensuse, next time. </remark>
 
+ <sect1 xml:id="sec.security.vulnerabilities">
+  <title>Software Vulnerabilities</title>
   <para>
-   To handle security competently, it is important to observe some
-   recommendations. You may find the following list of rules useful in
-   dealing with basic security concerns:
+   Software vulnerabilities are a class of software bugs that can be
+   exploited by attackers to obtain unauthorized access to data or use
+   attacked systems in unintended ways. Especially for remote services,
+   for example HTTP servers, vulnerabilities can be very critical. In
+   current computer environments, vulnerabilities have to be expected.
+   Usually it is the obligation of programmers to fix these issues when
+   they become known. It is then the administrators task to quickly and
+   safely install provided updates to protect the systems.
```

Comment:
Suggestion for para: `<para>Software vulnerabilities are issues in software that can be exploited to obtain unauthorized access or misuse systems. Vulnerabilities are especially critical if they affect remote services, such as HTTP servers. Computer systems are very complex, therefore, they always include certain vulnerabilities.</para><para>When such issues become known, they must usually be fixed in the software by software developers. The resulting update must then be installed by system administrators in a timely and safe manner on affected systems.</para>`